modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,274 @@
+<4> Specify the DPDK image that contains your application and the DPDK library used by application.
+<5> Removing all capabilities (`ALL`) from the container's securityContext means that the container has no special privileges beyond what is necessary for normal operation.
+<6> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access.
+<7> Mellanox network interface controller (NIC) requires the `NET_RAW` capability.
+<8> Specify the user ID used for running the container.
+<9> This setting indicates that the container or containers within the pod should not be granted privileged access to the host system.
+<10>  This setting allows a container to escalate its privileges beyond the initial non-root privileges it might have been assigned.
+<11> This setting ensures that the container runs with a non-root user. This helps enforce the principle of least privilege, limiting the potential impact of compromising the container and reducing the attack surface.
+<12> Mount a hugepage volume to the DPDK pod under `/mnt/huge`. The hugepage volume is backed by the emptyDir volume type with the medium being `Hugepages`.
+<13> Optional: Specify the number of DPDK devices allocated for the DPDK pod. If not explicitly specified, this resource request and limit is automatically added by the SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by SR-IOV Operator. It is enabled by default and can be disabled by setting the `enableInjector` option to `false` in the default `SriovOperatorConfig` CR.
+<14> Specify the number of CPUs. The DPDK pod usually requires exclusive CPUs to be allocated from the kubelet. This is achieved by setting CPU Manager policy to `static` and creating a pod with `Guaranteed` QoS.
+<15> Specify hugepage size `hugepages-1Gi` or `hugepages-2Mi` and the quantity of hugepages that will be allocated to the DPDK pod. Configure `2Mi` and `1Gi` hugepages separately. Configuring `1Gi` hugepage requires adding kernel arguments to Nodes. For example, adding kernel arguments `default_hugepagesz=1GB`, `hugepagesz=1G` and `hugepages=16` will result in `16*1Gi` hugepages be allocated during system boot.
++
+[NOTE]
+====
+Modify the configuration of your DPDK application to run with user ID `1001` and group ID `1001`. Depending on the specific requirements and configuration options of your DPDK application, you can accomplish this in several ways.
+====
++
+<16> If your performance profile is not named `cnf-performance profile`, replace that string with the correct performance profile name.
+--
++
+. Create the DPDK pod by running the following command:
++
+[source,terminal]
+----
+$ oc create -f dpdk-pod-rootless.yaml
+----
+
+. Get detailed information about the `dpdk-app` within the `test-namespace` by running the following command:
++
+[source,terminal]
+----
```

Comment:
what I'm missing here is an example of how to run dpdk-testpmd. So, we could say that our dpdk-base image has already the required capabilities for dpdk-testpmd and that we could run it with the following command.

@kononovn, please share the command you use in your tests. Just to receive traffic from the vf and forward it to the tap device.

Reply:
I am not entirely following the context here, do I need to make some reference to `dpdk-testpmd` with respect to this example.